modules/tinymce/src/core/test/ts/browser/delete/BlockMergeBoundaryTest.ts

```diff
@@ -72,24 +72,6 @@ describe('browser.tinymce.core.delete.BlockMergeBoundary', () => {
       const boundaryOpt = readBlockBoundary(true, [ 1, 0 ], 1);
       assertBlockBoundaryNone(boundaryOpt);
     });
-
-    it('TINY-10590: If the cursor is between an internal block and a text-node it should not count as a block boundary', () => {
-      setHtml('<div>A<p>B</p>C</div>');
-      const boundaryOpt = readBlockBoundary(false, [ 0 ], 2);
-      assertBlockBoundaryNone(boundaryOpt);
-    });
-
-    it('TINY-10590: If the cursor is in a text node and attempt to go outside of it, going backwards, it should not count as a boundary', () => {
-      setHtml('<div>A<p>B</p>C</div>');
-      const boundaryOpt = readBlockBoundary(false, [ 0, 1 ], 0);
-      assertBlockBoundaryNone(boundaryOpt);
-    });
-
-    it('TINY-10590: If the cursor is in a text node and attempt to go outside of it, going forwards, it should not count as a boundary', () => {
-      setHtml('<div>A<p>B</p>C</div>');
-      const boundaryOpt = readBlockBoundary(true, [ 0, 1 ], 1);
-      assertBlockBoundaryNone(boundaryOpt);
-    });
```

Comment:
is that correct that this behavior is now different?

Reply:
Yes. These were tests for the previous solution, which relied upon just ignoring this situation and relying on the browser to fix it. This was not what we wanted, so this was not the correct behavior.